frontend/src/components/WeaponInvetory.vue

```diff
@@ -0,0 +1,727 @@
+<template>
+  <div
+    class="weapon-icon"
+    v-bind:class="[(getWeaponDurability(weapon.id) === 0 ? 'no-durability' : '')]"
+    v-tooltip="{
+      content: tooltipHtml ,
+      trigger: (isMobile() ? 'click' : 'hover')
+    }"
+    @mouseover="hover = !isMobile() || true"
+    @mouseleave="hover = !isMobile()"
+  >
+
+    <div class="loading-container" v-if="!allLoaded">
+      <i class="fas fa-spinner fa-spin"></i>
+    </div>
+
+    <!-- adventure selected weapon display -->
+    <div class="displayed-weapon" v-if="displayType == 'adventure' && isMobile()">
+        <div :class="'weapon-img img-adventure frame-'+ (weapon.stars || 0)">
+          <!-- WEAPON ID -->
+          <div class="id">{{$t('weaponIcon.id')}} {{ weapon.id }}</div>
+
+          <!-- below use of weapon.id is for test purpose, should be replaced with getWeaponCosmetic(weapon.id) -->
+          <img v-if="showPlaceholder" v-bind:class="showCosmetics ? ' weapon-cosmetic-applied-'
+          + getWeaponCosmetic(weapon.id) : ''"
+            class="placeholder" :src="getWeaponArt(weapon)" />
+          <!-- element icon -->
+          <span :class="weapon.element.toLowerCase() + '-icon'"></span>
+        </div>
+    </div>
+
+    <!-- inventory selected weapon list display -->
+    <div v-if="displayType == 'inventory' || !isMobile()" class="glow-container" ref="el" :class="['glow-' + (weapon.stars || 0)]">
+      <div class="weapon-flex">
+        <div :class="'weapon-img frame-'+ (weapon.stars || 0)">
+            <!-- WEAPON ID -->
+          <div class="id">{{$t('weaponIcon.id')}} {{ weapon.id }}</div>
+          <!-- below use of weapon.id is for test purpose, should be replaced with getWeaponCosmetic(weapon.id) -->
```

Comment:
Is this relevant? Also file name has a typo. Maybe tidy up the html here too. Inconsistent spacing